vm-mock/vm.js

```diff
@@ -5,8 +5,9 @@ const { createHash } = await import('node:crypto');
  */
 
 // Those both addresses have been randomly generated
-const callerAddress = 'A12UBnqTHDQALpocVBnkPNy7y5CndUJQTLutaVDDFgMJcq5kQiKq';
-const contractAddress = 'A12BqZEQ6sByhRLyEuf0YbQmcF2PsDdkNNG1akBJu9XcjZA1eT';
+const callerAddress = 'AU12UBnqTHDQALpocVBnkPNy7y5CndUJQTLutaVDDFgMJcq5kQiKq';
+const contractAddress = 'AS12BqZEQ6sByhRLyEuf0YbQmcF2PsDdkNNG1akBJu9XcjZA1eT';
+const addressBytesLength = 53;
```

Comment:
I though that addresses are not fix-sized. If it true and if here you intentionally simplified by fixing the length to 53, then it's fine, just add a comment to explain this.

Reply:
In that case it was just for me to check something. @AurelienFT what would be a true test case, I did not understand how it works in the rust?

Reply:
The address is PREFIX+base58encode(varint version, bytes)

Reply:
It's not always 53